actionview/lib/action_view/view_paths.rb

```diff
@@ -72,6 +72,7 @@ def local_prefixes
 
     # :stopdoc:
     @all_view_paths = {}
+    @@dynamic_view_paths = Concurrent::Map.new
```

Comment:
nitpick: are we able to use a class instance variable instead of an class variable here?